src/components/date/date.js

```diff
@@ -506,6 +506,7 @@ class Date extends React.Component {
     // TODO: Pull datepicker into own component to wrap third party
     const datePicker = this.state.open ? this.renderDatePicker() : null;
 
+    /* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
whats this been added for?

Reply:
This rule can be disabled here as the element has a presentational role. This bug is resolved in the master branch of `eslint-plugin-jsx-a11y` but is not yet released

Reply:
This was noted in the commit message, but I can add as a code comment too.